bouture.js

```diff
@@ -59,30 +60,42 @@ function completeElement (tag, args) {
         Object.keys(arg)
           .forEach(attributeName => {
             const attributeValue = arg[attributeName]
-            switch (typeof attributeValue) {
-              case 'boolean':
-                if (attributeValue) {
-                  element.setAttribute(attributeName, '')
+            // check for event binding
+            const eventName = attributeName.match(/^on/) && attributeName.replace(/^once|on/, '')
+            if (eventNames.has(eventName)) {
+              if (typeof attributeValue === 'function') {
+                const options = {}
+                if (attributeName.match(/^once/)) {
+                  options.once = true
```

Comment:
This works. Another possibilty is `const options = {once: attributeName.match(/^once/)}`. I'll let you decide what you prefer

Reply:
More concise  - Thx